.gitattributes

```diff
@@ -1 +1 @@
-*.js eol=lf
```

Comment:
Please re-add this as well. Did not realize that this file was non-empty before.